hwi/aladin_crawling.py

```
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### 1. 라이브러리 임포트
+
+# In[108]:
+
+
+get_ipython().system('pip install bs4')
```

Ruff (0.13.1)

Undefined name `get_ipython`

(F821)